caluma/caluma_core/events.py

```diff
@@ -0,0 +1,28 @@
+import logging
+
+from django.dispatch import receiver as on  # noqa
+
+logger = logging.getLogger(__name__)
+
+
+def send_event(signal, **kwargs):
+    """
+    Send events and handle exceptions in receiver functions.
+
+    This wrapper handles the sending of signals as well as handling of any Exceptions that
+    occur. Exceptions are logged with `ERROR` level, but not raised, in order to prevent
+    rollback of the transaction.
+    """
+    result = signal.send_robust(**kwargs)
+    for func, exc in result:
+        if exc is not None:
+            logger.error(
```

Comment:
You could use `logger.exception()` instead, which produces some more output

Reply:
From the [docs](https://docs.python.org/2/library/logging.html#logging.Logger.exception):

```
This method should only be called from an exception handler.
```

So for using this we would have to raise the exception, then catch it and then log it.

Reply:
Right! Let's keep it as is then